js/foundation.toggler.js

```diff
@@ -59,12 +59,19 @@ class Toggler extends Plugin {
     }
 
     // Add ARIA attributes to triggers
-    var id = this.$element[0].id;
-    $(`[data-open="${id}"], [data-close="${id}"], [data-toggle="${id}"]`)
-      .attr({
-        'aria-controls': id,
-        'aria-expanded': this.$element.is(':hidden') ? false : true
+    var id = this.$element[0].id,
+        $triggers = $(`[data-open~="${id}"], [data-close~="${id}"], [data-toggle~="${id}"]`),
+        that = this;
+
+    $triggers.each(function(index, element) {
+      var $this = $(element);
+
+      //make sure not to duplicate id in aria-controls
+      $this.attr({
+        'aria-controls': `${$this.attr('aria-controls') ? $this.attr('aria-controls').replace(id, '') : ''} ${id}`.trim(),
```

Comment:
This would actually remove `test1` inside `id="test15 test25"` and change it to `id="5 test25 test1"`. You should use RegExp instead with the boundary special character `\b`.